docs/api_reference/src/quantities.tex

```diff
@@ -902,13 +899,30 @@
 \end{codeblock}
 \end{example}
 
+\rSec2[qty.expr.temp.concepts]{Concepts}
+
+\begin{itemdecl}
+template<typename T>
+concept @\defexposconceptnc{SymbolicArg}@ = (!std::is_const_v<T>) && (!std::is_reference_v<T>);  // \expos
+
+template<typename T>
+concept @\defexposconceptnc{SymbolicConstant}@ = @\exposconceptnc{SymbolicArg}@<T> && std::is_empty_v<T> &&         // \expos
+                           std::is_trivial_v<T> && std::@\stdconcept{semiregular}@<T> && std::is_final_v<T>;
+\end{itemdecl}
+
+\pnum
+The concept \exposconcept{SymbolicConstant}
+is used to constrain the program-defined types
+that derive from the library types
+that represent the parts that make up a quantity.
```

Comment:
I am not sure if that is correct. Are `si::metre` or `isq::length` program-defined types?

I would more scope on what those are used for: "types used in/passed to expression templates".

Reply:
The statement is correct.
It just isn't the complete statement
of what a _`SymbolicConstant`_ is or is used for.